src/KeyCommand.web.js

```diff
@@ -0,0 +1,109 @@
+import _ from 'underscore';
+import Events from 'events';
```

Comment:
@luacmartins Are we comfortable with this lib?

Reply:
any thoughts @luacmartins

Reply:
Apologies for the late response, I was ooo for the past 15 days. Yes, `events` seems fine and it's a standard lib that seems to be included by either node or webpack. From their docs:

>You usually do not have to install events yourself! If your code runs in Node.js, events is built in. If your code runs in the browser, bundlers like [browserify](https://github.com/browserify/browserify) or [webpack](https://github.com/webpack/webpack) also include the events module.